CODEOWNERS

```diff
@@ -3,11 +3,27 @@
 *         @dconnolly @FiloSottile @str4d
 
 # Spec maintainers own each individual spec.
-age.md          @FiloSottile
-age-plugin.md   @str4d
-vrf-r255.md     @FiloSottile
-poseidon-*.md   @str4d
-jq255.md        @pornin
-jq255/          @pornin
-chacha8rand.md  @FiloSottile @rsc
-chacha8rand/    @FiloSottile @rsc
+age.md                  @FiloSottile
+age-plugin.md           @str4d
+chacha8rand.md          @FiloSottile @rsc
+chacha8rand/            @FiloSottile @rsc
+https-bastion.md        @FiloSottile @rgdd
+https-bastion/          @FiloSottile @rgdd
+jq255.md                @pornin
+jq255/                  @pornin
+poseidon-*.md           @str4d
+signed-note.md          @FiloSottile @rsc
+signed-note/            @FiloSottile @rsc
+streaming-encryption.md @FiloSottile @str4d
+streaming-encryption/   @FiloSottile @str4d
+sunlight.md             @FiloSottile
+sunlight/               @FiloSottile
+tlog-checkpoint.md      @FiloSottile @rgdd
+tlog-checkpoint/        @FiloSottile @rgdd
+tlog-cosignature.md     @FiloSottile @rgdd
+tlog-cosignature/       @FiloSottile @rgdd
+tlog-witness.md         @FiloSottile @rgdd
+tlog-witness/           @FiloSottile @rgdd
```

Comment:
These specs along with `https-bastion` will also have a maintainer from the Google Trust Fabric team per https://github.com/C2SP/C2SP/issues/43#issuecomment-1894178899. If that maintainer is decided before this PR is merged, I'll rebase to include them.

Reply:
Timing out on this. A subsequent PR can add this maintainer when they are decided on.

Reply:
The additional maintainers are added in #57.